cypress/local-only/tests/integration/trialSession/trial-session-paper-pdf.cy.ts

```diff
@@ -120,6 +120,10 @@ describe('Trial Session Paper Pdf', { scrollBehavior: 'center' }, () => {
           );
           cy.visit(`/edit-trial-session/${createdTrialSessionId}`);
           cy.get('[data-testid="trial-session-judge"]').select('Colvin');
+          cy.get('[data-testid="trial-session-judge"] option:selected').should(
```

Comment:
Sometimes the selection hadn't actually finished being loaded in before the test continued.